### PR TITLE
Test on higher php versions

### PR DESCRIPTION
**Pull request by BackEndTea**

### CODE REVIEW

**Comment by MKodde** [comment]
Welcome @BackEndTea thanks for your contribution!

The 5.6 build dies on a lack of memory during the installation of the composer dependencies. Maybe experiment with adding `COMPOSER_MEMORY_LIMIT=-1` to the composer install command in the ant build.xml. Just thinking out loud. Another solution might be to allocate more memory to travis if they allow that.

File: bootstrap.php [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?php
+
```
Comment (MKodde) [inline bootstrap.php]: Please add the copyright statement in this file, as we do in other PHP files.

File: phpunit.xml [hunk 2]
```diff
@@ -9,7 +9,7 @@
          processIsolation="false"
          stopOnFailure="false"
          syntaxCheck="false"
-         bootstrap="vendor/autoload.php">
+         bootstrap="bootstrap.php">
```
Comment (MKodde) [inline phpunit.xml]: Is using the vendor/autoload.php no longer required?

Comment (BackEndTea) [reply phpunit.xml]: We do use it, but its included in the bootstrap.php

**Comment by BackEndTea** [comment]
Thank you @MKodde

**Pull request merged**